jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,838 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+Nim version 1.6 is now officially released!
+
+A year in the making, 1.6 is the latest stable release and by far the largest yet.
+We're proud of what we, the core team and dedicated volunteers, have accomplished with this milestone.
+Here are some stats:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in >40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and last 2 months it entered the top-50 (https://forum.nim-lang.org/t/8297).
+We hope this release will reinforce this trend, building on Nim's core strengths:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; LISP-like flexibility; strong C, C++, JS, Python interop;
+and best-in class metaprogramming.
```

Comment:
thinking of something like this, improvements/suggestions welcome:

Did we mention fast compile times (8s to recompile nim vs XXX for rust), compiles and runs wherever you have a C compiler [gameboy kickstarter](https://forum.nim-lang.org/t/8375), and yes, even write [malware](https://thehackernews.com/2021/03/researchers-spotted-malware-written-in.html).

Reply:
I think the Go program has fewer system dependencies so may not be a fair comparison.

Reply:
IMO it's better to avoid mentioning malware, but that's just my view

Reply:
2MB is pretty fair given that's caused by Go's language design. Go's reflection mechanism means you cannot optimize away unused routines.

Reply:
@deech even if you link nim hello world binary statically, it'll surely be under 1MB, probably under 200KB

Reply:
I agree, let's not mention malware, especially since many people are getting false positives alerts about Nim being malware...

About the rest, it sounds weirdly worded. You should instead just have a section where exciting projects are mentioned, in particular the kickstarter.

Reply:
> I agree, let's not mention malware

yes, this was a long shot; I'm not mentioning it.

I've added a section "Why use Nim?", please provide feedback